vscode-antimony/src/server/stibium/stibium/tree_builder.py

```diff
@@ -70,8 +86,11 @@
 }
 
 OPERATORS = {'EQUAL', 'COLON', 'ARROW', 'SEMICOLON', 'LPAR', 'RPAR', 'STAR', 'PLUS', 'MINUS',
-             'DOLLAR', 'CIRCUMFLEX', 'COMMA', 'SLASH', "AEQ", "DBLQUOTE"}
-KEYWORDS = {'ANNOT_KEYWORD', 'IN', 'MODEL', 'FUNCTION', "UNIT", "HAS", "IS", "SUBSTANCEONLY"}
+             'DOLLAR', 'CIRCUMFLEX', 'COMMA', 'SLASH', "AEQ", "DBLQUOTE",
+             'MORETHAN', '__ANON_1', 'LESSTHAN', '__ANON_0', '__ANON_2', '__ANON_3',

```

Comment:
I keep commenting on these anon's on the different PRs.  They definitely wouldn't merge properly!  They should be given proper names.